src/main/java/org/folio/services/ftp/UploadService.java

```diff
@@ -1,11 +0,0 @@
-package org.folio.services.ftp;
-
-
-import io.vertx.core.Context;
-import io.vertx.core.Future;
-
-public interface UploadService {
```

Comment:
Why do you think we don't need interface here? Basically we have some contract and 2 implementations: upload to ftp and sftp

Reply:
I see the difference now, resolving comment